src/Microsoft.Azure.WebJobs.Logging/FunctionInstanceLogItem.cs

```diff
@@ -156,7 +156,8 @@ public void Validate()
             {
                 if (this.StartTime > this.EndTime)
                 {
-                    throw new InvalidOperationException("End Time must be greater than start time");
+                    // This can happen in rare cases if the clock adjusts while the function is running. 
+                    this.EndTime = this.StartTime.AddMilliseconds(1);                    
```

Comment:
Do we prefer this over removing the validation check altogether? I suppose the dashboard or other consumers might throw up if they see endtime < starttime.